public_html/assets/css/styles.css

```diff
@@ -78,18 +78,13 @@ input:focus {background-color:transparent;}
 /* ---------------------------------
     HOME BG (z-index: -13, -12, -11)
 ------------------------------------ */
-.bg-imges-fade,
-.bg-images-rotating-banner {position:fixed; top:0; left:0; width:100%; height:100%; z-index: -20;}
-.bg-img {position:absolute; top:0; left:0; bottom:0; width:100%; background:#fff none center center no-repeat; background-size:cover;}
-.bg-img--fade.bg-img--over {background-image:url(../img/BGS/bg-bw.jpg); z-index:-11;}
-.bg-img--fade.bg-img--under {background-image:url(../img/BGS/bg-color.jpg); z-index:-12;}
-
-.bg-img--rotating-banner {z-index:-13; background-size:contain;}
-.bg-img--rotating-banner.bg-img--active {z-index:-11;}
-.bg-img--rotating-banner.bg-img--prev {z-index:-12;}
+.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20;}
+.bg-img {position:absolute; top:0; left:0; bottom:0; width:100%; background:#fff none center center no-repeat; z-index:-13; background-size:contain;}
+.bg-img.bg-img--active {z-index:-11;}
+.bg-img.bg-img--prev {z-index:-12;}
 
 @media screen and (max-width:959px) and (orientation:portrait), screen and (max-width:767px) {
-    .bg-img.bg-img--rotating-banner {background-image:none;}
```

Comment:
i'm removing support for the old opacity changing background, so this can be simlified